src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,169 @@
+package kitchenpos.application;
+
+import kitchenpos.dao.MenuDao;
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.dao.MenuProductDao;
+import kitchenpos.dao.ProductDao;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class MenuServiceTest {
+
+    private Menu 강정치킨;
+    private Menu 양념치킨;
+    private Menu 후라이드치킨;
+    private Product 상품;
+    private List<Menu> 메뉴리스트 = new ArrayList<>();
+    private List<MenuProduct> 상품리스트 = new ArrayList<>();
+
+    private MenuService menuService;
+
+    @Mock
+    private MenuDao menuDao;
+    @Mock
+    private MenuGroupDao menuGroupDao;
+    @Mock
+    private MenuProductDao menuProductDao;
+    @Mock
+    private ProductDao productDao;
+
+    @BeforeEach
+    public void setUp() {
+        상품 = new Product("테스트상품",new BigDecimal(20000));
+        상품리스트.add(new MenuProduct(2L,1));
+        상품리스트.add(new MenuProduct(3L,1));
+        강정치킨 = new Menu("강정치킨", new BigDecimal(17000),1L, 상품리스트);
+        양념치킨 = new Menu("양념치킨", new BigDecimal(18000),2L, 상품리스트);
+        후라이드치킨 = new Menu("후라이드치킨", new BigDecimal(16000),3L, 상품리스트);
+        메뉴리스트.add(강정치킨);
+        메뉴리스트.add(양념치킨);
+        메뉴리스트.add(후라이드치킨);
+        menuService = new MenuService(menuDao, menuGroupDao, menuProductDao, productDao);
+    }
+
+    @DisplayName("메뉴 등록 테스트")
+    @Test
+    void createMenu() {
+        when(menuGroupDao.existsById(any())).thenReturn(true);
+        when(productDao.findById(any())).thenReturn(Optional.ofNullable(상품));
+        mockSaveMenu(강정치킨);
+        mockSaveMenuProduct(상품리스트.get(0));
+
+        Menu resultMenu = menuService.create(강정치킨);
+
+        assertAll(
+                () -> assertThat(resultMenu.getName()).isEqualTo(강정치킨.getName()),
```

Comment:
```suggestion
                () -> assertThat(resultMenu.getName()).isEqualTo("강정치킨"),
```

mock 처리를 해서 resultMenu와 강정치킨 객체가 동일하니 같은 객체의 속성 비교하면 로직이 잘못 되더라도 무조건 테스트 통과로 나오지 않을까요?